testsite/requirements.txt

```diff
@@ -1,10 +1,13 @@
 # testsite for djaodjin-extended-templates
 beautifulsoup4==4.11.1            # 4.11.1 compatibles with Py3.10
-Django==3.2.25
-djangorestframework==3.14.0       # 3.12.4 not compatible with Django4.2
-django-assets==0.12
+Django==3.2.24 ; python_version < "3.9"
```

Comment:
Django 3.2.25 is the latest 3.2 version.

Reply:
Oops. Good catch.